components/Modals/NotePublishModal.tsx

```diff
@@ -221,8 +236,8 @@ function NotePublishModal({
     let params;
     if (publishedType === "UNPUBLISHED") {
       params = {
+        assign_doi: checkBoxTwo,
```

Comment:
Can we change to something like `checkBoxDOI` or something like that? I think if we switch the order of checkboxes this may cause a logic bug

Reply:
Good point, I'll update this.